response_operations_ui/forms.py

```diff
@@ -495,3 +512,15 @@ class BannerEditForm(FlaskForm):
     )
     banner_id = StringField("id")
     delete = BooleanField("Delete banner", default=False)
+
+
+class Form(FlaskForm):
```

Comment:
Why does this exist?  Can't you have the OptionsForm inherit from FlaskForm, then add a validate function in there?